index.html

```diff
@@ -1,111 +1,424 @@
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+        </div>
+      </section>
+      <section class="faqs">
+        <div>
+          <h1>FAQs</h1>
+        </div>
+        <div>
+          <div>
+            <h1>What are the NTFs?</h1>
+            <button>+</button>
+          </div>
+          <p>
+            Urna vitae erat et lacus, consectetur ac nulla vestibulum lobortic.
+            Nula dapibus urna volutpat vestibulum lobortic. Nulla dapibus urna
+            volutpat venenatis, risus faucibus.
+          </p>
+          <div>
+            <h1>How do i get NTFs?</h1>
+            <button>+</button>
+          </div>
+          <div>
+            <h1>How can we buy your NTFs?</h1>
+            <button>+</button>
+          </div>
+          <div>
+            <h1>Who are the team behind the project?</h1>
+            <button>+</button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <div class="logo">
+          <a class="logo" href="#">Binabox</a>
+          <p>We are the best way to check the rarity of NFT collection.</p>
+        </div>
+        <div class="menu">
+          <ul>
+            <h1>
+              <a href="#" class="menu">Subscribe</a>
+            </h1>
+            <li>
+              <a href="#" class="menu">Home</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Blog</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Catalog</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Contact us</a>
+            </li>
+          </ul>
+          <ul>
+            <h1>
+              <a href="#" class="menu">Company</a>
+            </h1>
+            <li>
+              <a href="#" class="menu">Terms Canditions</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Privacy Policy</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Whitepaper</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Help Center</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div>
+        <p>BINABOX 2022 - ALL rights reserved</p>
+        <span class="icon">Twitter</span>
```

Comment:
The following elements are also better combined into a list and made into links

Reply:
done